base/deprecated.jl

```diff
@@ -612,6 +612,21 @@ end
 const MS_ASYNC = 1
 const MS_INVALIDATE = 2
 const MS_SYNC = 4
+
+for constant in (:MS_ASYNC, :MS_INVALIDATE, :MS_SYNC)
+    constant_str = string(constant)
+    @eval begin
+        @doc """
+            $($constant_str)
+
+        Enum constant for [`msync`](:func:`msync`). See your platform man page for details.
+        (not available on Windows).
+        """ ->
+        $constant
+    end
+end
```

Comment:
Actually, that's probably a bug in `genstdlib`, since the docstrings are the same object rather than different docstrings that happen to have the same signature. I'll try track that one down.